include/tvm/tir/schedule/schedule.h

```diff
@@ -779,29 +779,33 @@ class Schedule : public runtime::ObjectRef {
    * \param debug_mask Do extra correctness checking after the class creation
    * and each time after calling the Replace method.
    * \param error_render_level The level of error rendering
+   * \param enable_check Whether to enable prequisite checks for schedule primitives.
```

Comment:
Please add comments saying that it's user's responsibility to ensure the correctness if we disable the checks

Reply:
Sounds good, will do it.